### PR TITLE
Improved german translation.

### PR DESCRIPTION
**Pull request by yetipenguin**

### CODE REVIEW

**Comment by PhysSong** [comment]
Hi @yetipenguin, thank you for the contribution.
If you're interested in improving translations, please join https://www.transifex.com/lmms/lmms/.

File: data/locale/de.ts [hunk 1]
```diff
@@ -5735,7 +5735,7 @@ PM bedeutet Phasen-Modulation: Die Phase von Oszillator 3 wird durch Oszillator
     </message>
     <message>
         <source>Timeline controls</source>
-        <translation>Zeitlinien Regler</translation>
+        <translation>Zeitlinienregler</translation>
```
Comment (JohannesLorenz) [inline data/locale/de.ts]: The better translation for timeline would be "Zeitstrahl", "Zeitachse" or "Zeitleiste". Also, the [Duden](https://www.duden.de/sprachwissen/rechtschreibregeln/bindestrich) recommends a minus between composed words to increased readability.

Asides, this translation occurs twice in that file. Can you please fix both?

**Comment by zonkmachine** [comment]
These changes should be proposed on https://www.transifex.com/lmms/lmms/ (and this PR closed)
As I understand it, if we merge this PR the changes will be reverted when we merge the changes from our Transifex account.

**Comment by yetipenguin** [comment]
It's  pretty annoying to use an external service to change language strings inside of a software. But anyway, I joint, now I need someone to accept the request.

@JohannesLorenz I agree with you on "Zeitstrahl" but would not opt for a hypen in between "Zeitstrahl" and "Regler". The Duden says you can but a hypen, but you don't need to.
In my opinion, a hyphen makes the string less readable to a german reader and also make the string longer.
If you insist on the hyphen I will add it nonetheless. :-)

**Comment by JohannesLorenz** [comment]
Np, I don't resist on it.

Feel free to close the PR after you had success on Transifex.

**Comment by yetipenguin** [comment]
@JohannesLorenz and @zonkmachine I'm still waiting to get my application for the lmms team on Transifex approved. Do I need to contact someone actively or just wait?

**Comment by JohannesLorenz** [comment]
> I'm still waiting to get my application for the lmms team on Transifex approved. 

For me, it took ~ 3 months. Maybe @liushuyu knows how to speed things up?

**Comment by yetipenguin** [comment]
Thanks everyone! :-) I have access to Transifex and changed the strings.

**Pull request closed without merging**